pkgs/development/dotnet-modules/python-language-server/create_deps.sh

```diff
@@ -0,0 +1,36 @@
+#! /usr/bin/env bash
+#! nix-shell -i python dotnet-sdk_3
```

Comment:
```suggestion
#! /usr/bin/env nix-shell
#! nix-shell -p python3 dotnet-sdk_3 -i bash
```

The above looks right, but it was mostly done off the top of my head... I have this link bookmarked, which might help troubleshoot if I'm actually wrong: https://gist.github.com/travisbhartwell/f972aab227306edfcfea. (Note: `-i` is for the interpreter -- this is a bash script, so it should be bash and not python.)

You can test this by running the script as you normally would: `./create_deps.sh path/to/ms/py/lsp/src`. If it works, it's probably right :P

Reply:
Seems to work